Improve view on mobile devices

sfdeloach opened the pull request:
Issue #33 

This PR started from scratch and was more careful to preserve the style of the original website.

On a desktop, this PR should look the same as the existing website except for two items: (1) The top navbar text and GitHub symbol does not extend edge-to-edge (this is an easy fix if preferred), (2) the highlighted text when an API item is selected from the right window only highlights the text, the highlight does not extend across the entire anchor element (this one was a challenge for me since padding the top was the only solution I could come up with so that the item was not hidden under the top nav bar):

![Screenshot from 2022-01-24 16-44-56](https://user-images.githubusercontent.com/15959673/150873566-9a001ee0-ee37-4920-b8f7-28e1b8a3558f.png)

The mobile views should look the same as the last PR, except I changed the bottom nav bar to look more like the existing website:

![Screenshot from 2022-01-24 16-45-50](https://user-images.githubusercontent.com/15959673/150874257-af216cb0-72d5-4de5-a88d-1f8b27ade757.png)

![Screenshot from 2022-01-24 16-46-14](https://user-images.githubusercontent.com/15959673/150874293-b74a056d-3dea-4506-a5fe-319da10d96c5.png)

joaodasilva commented:
Many, many thanks for these changes -- the site looks so much better!

Just a couple of smaller tweaks left. If you don't have the time then I'd go ahead and merge as-is and then do these fixes afterwards. Let me know if you want to fix them in this PR.

**1.** the frontpage shows the main h1 too close to the top navbar. Maybe leave a line of spacing there.

![snip1](https://user-images.githubusercontent.com/1609375/151056994-837dda3d-10d8-4201-87dc-1616761801ed.png)



**2.** the navbar to the right seems to be missing an empty line spacing between he first section and the second.

![snip3](https://user-images.githubusercontent.com/1609375/151057417-d3ec0f2d-871a-4de1-98e8-e714a440fffc.png)
![snip4](https://user-images.githubusercontent.com/1609375/151057528-851df0c1-928e-4e13-8e50-42eab3b9202c.png)


**3.** the links in the bottom navbar on mobile are not aligned. Compare the before and after pics:

![snip5-before](https://user-images.githubusercontent.com/1609375/151057614-19acdc0f-0686-4475-ae03-62534729bf68.png)
![snip5-after](https://user-images.githubusercontent.com/1609375/151057627-8fc17344-1753-44ad-b91f-9d53aba41802.png)

**4.** text seems to be too close to the bottom of tables, maybe add a margin-bottom to tables?

![snip6](https://user-images.githubusercontent.com/1609375/151057800-2547642c-debb-432c-8789-713a2458a100.png)

sfdeloach commented:
Fixed four items in issue windowjs#73:

1. More space added on frontpage between span.title and navbar (this section is not wrapped in an h1 element, an extra break was added to create space)

![001](https://user-images.githubusercontent.com/15959673/151095647-3b9e5c5a-6503-4533-ac39-db01b75b2c9d.png)

2. Space added on the right navbar between the first and second sections

![002](https://user-images.githubusercontent.com/15959673/151095664-db286de4-f462-4d03-8f93-9b1e4e57a3d6.png)

3. Links at the bottom of the mobile view are placed into a table to left-justify content in columns (placed in two columns to provide extra space)

![003](https://user-images.githubusercontent.com/15959673/151095674-50f1cb41-598e-4193-9fd8-d74287f80c8b.png)

4. Adjusted margins for paragraphs that follow tables, creates more space.

![004](https://user-images.githubusercontent.com/15959673/151095690-5fb85480-a366-4f9a-bee1-66171bd05dc6.png)

joaodasilva commented:
It's live, and looking great! Thanks @sfdeloach for improving the site so much, especially on mobile.

sfdeloach commented:
My pleasure! Glad to help!
